pkg/core/options.go

```diff
@@ -30,49 +32,56 @@ var (
 )
 
 // ConcurrentList sets the max level of concurrency to retrieve core objects. It defaults to 2 x #cpus.
-func ConcurrentList(concurrentList int) Option {
+func ConcurrentList(v int) Option {
 	return func(s *Settings) {
-		if concurrentList == 0 {
+		if v == 0 {
 			s.concurrentList = defaultListConcurrency
 			return
 		}
-		s.concurrentList = concurrentList
+		s.concurrentList = v
 	}
 }
 
 // BatchSize sets the batch window to fetch core objects. It defaults to defaultBatchSize
-func BatchSize(batchSize int) Option {
+func BatchSize(v int) Option {
 	return func(s *Settings) {
-		if batchSize == 0 {
+		if v == 0 {
 			s.batchSize = defaultBatchSize
 			return
 		}
-		s.batchSize = batchSize
+		s.batchSize = v
```

Comment:
the rule of thumb to find a good identifier is that is should remain short and expressive. `v` is short, but not expressive. `theBatchSizeThatTheUserMayDefineFromAFlag` could be slightly more expressive, but definitely not short.
Please refrain here.

Reply:
i will revert the renaming, nonetheless, of course.

Reply:
ok we are good then